saskatoon/harvest/views.py

```diff
@@ -132,6 +134,21 @@ def get_context_data(self, **kwargs):
         context['cancel_url'] = reverse_lazy('harvest-detail', kwargs={'pk': self.object.pk})
         return context
 
+    def get_success_message(self, cleaned_data) -> str:
+        harvests_as_pickleader = self.object.pick_leader.person.harvests_as_pickleader
+        harvests_this_year = harvests_as_pickleader.filter(
+            start_date__year=timezone.now().date().year
+        )
```

Comment:
@mina-andajos you could use `.count()` after filtering, I think this would be more efficient than using `len()`

Reply:
here is relevant documentation https://docs.djangoproject.com/en/4.0/ref/models/querysets/#when-querysets-are-evaluated-1
see the Note in `len()` section

Reply:
Thanks for pointing this out